plans/upgrade.pp

```diff
@@ -1,18 +1,16 @@
 plan autope::upgrade(
-  TargetSpec                           $targets          = get_targets('peadm_nodes'),
-  String                               $version          = '2019.3.0',
-  String                               $ssh_user,
-  Enum['xlarge', 'large', 'standard']  $architecture     = 'xlarge',
-  Enum['google', 'aws', 'azure']       $provider         = 'google'
+  String                               $version             = '2021.0.0',
```

Comment:
we should probably use the latest available PE version as a default, currently 2021.2.0